packages/jest-types/src/Config.ts

```diff
@@ -41,6 +41,7 @@ export type DefaultOptions = {
   coveragePathIgnorePatterns: Array<string>;
   coverageReporters: Array<string | [string, any]>;
   coverageProvider: CoverageProvider;
+  crawlSymlinks: boolean;
```

Comment:
Can we call this `enableSymlinks` or move it into `haste` and call it `symlinks`?

Reply:
Oh yeah this should definitely go into `haste` because that will make it work with tools like Metro more easily.

Reply:
renamed to `enableSymlinks`